MOBILE_API.xsd

```diff
@@ -95,6 +95,7 @@
             <xs:attribute name="internal_name" type="xs:string"/>
             <xs:attribute name="rootscreen" type="xs:boolean"/>
             <xs:attribute name="since" type="xs:string"/>
+            <xs:attribute name="until" type="xs:string"/>
```

Comment:
`deprecated` attribute is missing from this element as well

Reply:
Also, it looks like the `removed` attribute is missing from all of these

Reply:
Addressed in https://github.com/smartdevicelink/rpc_spec/pull/224/commits/e8c90e49381d96065f6842b58c9e23b90ca0eb78